package.json

```diff
@@ -11,7 +11,9 @@
   "dependencies": {
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
+    "@mui/icons-material": "^5.8.4",
```

Comment:
yarn remove this package (b/c we don't need the hamburger icon or any other icons from this package right now)